src/test/munit/batch-read-write-test-suite.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:batch="http://www.mulesoft.org/schema/mule/batch"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+      xmlns:marklogic="http://www.mulesoft.org/schema/mule/marklogic"
+	xsi:schemaLocation="
+		http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+    http://www.mulesoft.org/schema/mule/batch http://www.mulesoft.org/schema/mule/batch/current/mule-batch.xsd
+		http://www.mulesoft.org/schema/mule/marklogic http://www.mulesoft.org/schema/mule/marklogic/current/mule-marklogic.xsd
+">
+
+  <!--  TODO - We probably need to read this information from a single file.-->
+  <marklogic:config name="MarkLogicConfig" configId="MarkLogicConfig">
+    <marklogic:connection
+        host="localhost"
+        port="8022"
+        username="mule2-test-user"
+        password="password"
+        authenticationType="DIGEST"/>
+  </marklogic:config>
+
+  <flow name="readTenDocsWriteTwoBatches" doc:id="63d4c67d-b038-4455-a4c1-f862de4bb735">
+    <marklogic:search-documents config-ref="MarkLogicConfig" collection="batch-input"/>
+    <batch:job jobName="BatchJob">
+      <batch:process-records>
+        <batch:step name="Step1">
+          <batch:aggregator size="5">
+            <marklogic:write-batch config-ref="MarkLogicConfig"/>
+          </batch:aggregator>
+        </batch:step>
+      </batch:process-records>
+    </batch:job>
+  </flow>
+
+	<munit:config name="batch-read-write-test-suite" />
+
+  <munit:test name="readTenDocsWriteTwoBatchesTest" doc:id="9e40fb4f-49b3-446d-8c08-da9e9ebb40bf" description="Test">
+		<munit:execution >
+			<flow-ref doc:name="Flow-ref to readTenDocsWriteTwoBatches" doc:id="283d79b2-275a-4f17-8dc2-4db74b4df159" name="readTenDocsWriteTwoBatches"/>
+		</munit:execution>
+		<munit:validation >
+			<munit-tools:sleep time="5" doc:name="Sleep" doc:id="079e4a6a-d159-4450-b2de-dfcd8a4fbd97" timeUnit="SECONDS"/>
+      <munit-tools:assert-equals doc:name="Assert equals" doc:id="157b01e7-5cd8-4555-a0bc-a7ba887e68c7" actual="#[payload.result.totalRecords]" expected="#[10]"/>
```

Comment:
Nice, this is good to see (though also wondering if the "doc" attributes can be removed).